src/utils/frame.js

```diff
@@ -97,3 +98,11 @@ export function formatDisplayName(frame: Frame) {
   displayName = simplifyDisplayName(displayName);
   return endTruncateStr(displayName, 25);
 }
+
+export function formatCopyName(frame: Frame) {
+  const displayName = formatDisplayName(frame);
+  const fileName = getFilename(frame.source);
+  const frameLocation = frame.location.line;
+
+  return `${displayName}  ${fileName}#${frameLocation}`;
```

Comment:
lets change this to `${displayName} (${fileName}#${frameLocation})` to be consistent with chrome and visually group the file name